### PR TITLE
Observability Pack for static websites

### PR DESCRIPTION
**Pull request by zstix**
## Summary
Adds some basic configuration for a static site, such as one created with [Gatsbyjs](https://www.gatsbyjs.com/). This pack includes:
* A synthetics monitor that will ping the website (with retries)
* Alert conditions for high JS errors and low apdex
* A troubleshooting dashboard

I took some guesses as to how we would handle variables. This was my first attempt at making a pack, I'd love some feedback!

### CODE REVIEW

**Comment by jpvajda** [comment]
adding to in review column so we can get this live. 🥳

File: packs/static-site/alerts/apdex-low.yml [hunk 1]
```diff
@@ -0,0 +1,24 @@
+name: End User Apdex (Low)
+
+details: Apdex below a specified amount for a duriation.
```
Comment (polfliet) [inline packs/static-site/alerts/apdex-low.yml:3]: typo: duration

File: packs/static-site/config.yml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+name: Static Site Pack
+
+description: Helpful configuration for anyone running a static website.
+
+level: New Relic
+
+icon: icon.jpeg
```
Comment (polfliet) [inline packs/static-site/config.yml:7]: Is there an icon that would fit well for this pack?
If not, can you add the newrelic icon and logo to this pack?

File: packs/static-site/config.yml [hunk 3]
```diff
@@ -0,0 +1,12 @@
+name: Static Site Pack
+
+description: Helpful configuration for anyone running a static website.
```
Comment (polfliet) [inline packs/static-site/config.yml:3]: Can we add a bit more detail to the description, something like this?
`This observability packs helps to monitor a static website. It uses the New Relic Browser agent to measure frontend performance. It also use New Relic Synthetics to measure uptime of the static site.`

Also some high-level instructions:
`In order to use this pack, make sure to update the Synthetics script URL variable to point to your website`

**Review by polfliet — COMMENTED** [review]
Great contribution Zack.
I had a few small comments and then it would be awesome to merge this pack in, thanks!

**Comment by polfliet** [comment]
Hi @bpeckNR! You have probably heard that we are moving to this new repository for observability packs (previously quickstarts). Here is a contribution from Zack that includes a Synthetic check and I would be great to get your review since you are our Synthetics expert! Thanks a lot!!

**Comment by zstix** [comment]
@polfliet I've made the requested updates. Let me know if you need anything else!

**Comment by polfliet** [comment]
thanks @zstix !

**Comment by zstix** [comment]
Unfortunately, I don't think the functionality needed by this work is ready yet. More specifically, we are running into the following issues:
* Browser alert condition types are not supported
* NRQL field is required on alert conditions (not relevant for browser alert conditions)
* No way to reference the static site's URL to ping (variable support)

For those reasons, this pack doesn't provide sufficient value and I'm going to close this PR. I just spun up a ticket to track the feature request for variable support (#52).

**Comment by bpeckNR** [comment]
> Hi @bpeckNR! You have probably heard that we are moving to this new repository for observability packs (previously quickstarts). Here is a contribution from Zack that includes a Synthetic check and I would be great to get your review since you are our Synthetics expert! Thanks a lot!!

I know this PR has been closed, but I wanted to circle back on the review that was requested. It looks like the sample Synthetic monitor included a few retries but would have needed to be a scripted API monitor. 

For a static site like this one, it may make sense to just use the built in Synthetics ping monitor type. While by default that uses a HEAD request, it can be configured to use a GET request and even validate the response.

As far as the retries built into the script, please note that the Synthetic environment includes 2 built in retries before a job shows up as failed. 

My suggestion is to simplify this and just use a ping type OOTB Synthetics monitor.

**Pull request closed without merging**